src/brogue/Items.c

```diff
@@ -1727,10 +1727,11 @@ short effectiveRingEnchant(item *theItem) {
     if (theItem->category != RING) {
         return 0;
     }
+
+    // Positive unidentified rings act as +1 until identified.
     if (!(theItem->flags & ITEM_IDENTIFIED)
         && theItem->enchant1 > 0) {
-
-        return theItem->timesEnchanted + 1; // Unidentified positive rings act as +1 until identified.
+        return 1; 
```

Comment:
I'm not sure this is fair to the player. If I enchant a known-good ring, I shouldn't have to wait until I identify it to gain the benefits of my enchant.